python/dgl/sparse/utils.py

```diff
@@ -0,0 +1,14 @@
+"""Utilities for DGL sparse module."""
```

Comment:
nit: is this sparse specific utils?

Reply:
So far it's sparse specific (really PyTorch specific).  Do you have a better place for it?